components/frontend-react/src/components/Appbar.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//MUI imports
+import {withStyles} from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Avatar from '@material-ui/core/Avatar';
+import TextField from '@material-ui/core/TextField';
+
+//Logo import
+import logo from '../logo.png';
+
+const styles = {
+  root: {
+    flexGrow: 1,
+    backgroundColor: '#ff5500'
+  },
+  grow: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginLeft: -12,
+    marginRight: 20,
+  },
+  avatar: {
+    margin: 10,
+  },
+  bigAvatar: {
+    margin: 10,
+    width: 60,
+    height: 60,
+  },
+};
+
+function BassaAppBar(props) {
+  const { classes } = props;
+
+  if (!props.isloggedIn) {
+  	return (
+	  <div className={classes.root}>
+	    <AppBar position="static" style={{background: '#625042'}}>
```

Comment:
Instead of '#color-code' using directly, store it as a variable and then use so that reusability increases

Reply:
Will fix this in next commit